index.php

```diff
@@ -52,6 +52,26 @@
                         <!-- výpis počtu zhlédnutí daného článku -->
                         |
                         Počet zhlédnutí: <?php echo $clanek["pocetZhlednutiText"]; ?>
+                        |
+                        <?php
+                        $slova = explode(" ", $clanek["obsah"]);
+                        $pocetSlov = count($slova);
+
+                        $dobaCteni = $pocetSlov / 200;
+                        $dobaCteniZaokrouhlena = ceil($dobaCteni);  // BONUS
+
+                        $minuty = 'min';
```

Comment:
`$minutyText` nebo tak neco? Podobne jsme to delali u toho poctu zhlednuti, aby si nemyslely, ze v ty promenny je ta hodnota, ale jenom ta jednotka :)